expense-splitter/src/components/friend/Friend.jsx

```diff
@@ -13,8 +14,8 @@ const Friend = () => {
   const [inputText, setInputText] = useState("");
 
   let inputHandler = (e) => {
-      var lowerCase = e.target.value.toLowerCase();
-      setInputText(lowerCase);
+    var lowerCase = e.target.value.toLowerCase();
+    setInputText(lowerCase);
```

Comment:
Would it not be better to use const and let instead of let and var?

```
const  inputHandler = (e) => {
    let lowerCase = e.target.value.toLowerCase();
    setInputText(lowerCase);
};

Reply:
Not my code and both are better off as const:
const inputHandler = (e) => {
    const lowerCase = e.target.value.toLowerCase();
    setInputText(lowerCase);
  };